src/Common/src/Interop/Linux/procfs/Interop.ProcFsStat.cs

```diff
@@ -331,28 +332,41 @@ internal static bool TryParseStatusFile(string statusFilePath, out ParsedStatus
                 switch (elements[0])
                 {
                     case "Pid":
-                        results.Pid = int.Parse(elements[1]);
+                        int.TryParse(elements[1], out results.Pid);
                         break;
```

Comment:
Don't we already have the pid?  Can we just remove this whole case?

Reply:
Can't. Need it for asserts.

E.g. ` Debug.Assert(!b || result.Pid == pid, "Expected process ID from status file to match supplied pid");`

Reply:
Done.